composer.json

```diff
@@ -8,7 +8,7 @@
     "prefer-stable": true,
     "require": {
         "php": "^8.1",
-        "doctrine/dbal": "^3.0",
+        "doctrine/dbal": "^4.0",
```

Comment:
@ro0NL I think we could remove doctrine/dbal
Now we just need doctrine/orm

Reply:
hehe, correct